api/admin/controller.py

```diff
@@ -633,10 +634,9 @@ def circulation_events(self):
         return dict({ "circulation_events": events })
 
     def bulk_circulation_events(self):
-        annotator = AdminAnnotator(self.circulation)
-
-        start = flask.request.args.get("start")
-        end = flask.request.args.get("end")
+        default = str(datetime.today()).split(" ")[0]
+        date = flask.request.args.get("date", default)
```

Comment:
it's a string. line 659 works timestamp comparisons hold when timestamps are cast to strings, e.g. `"2016-06-23" < "2016-06-23 09:00:00" < "2016-06-24"`.